test/Microsoft.AspNetCore.Mvc.Razor.ViewCompilation.FunctionalTests/ApplicationWithTagHelpersTest.cs

```diff
@@ -15,22 +18,27 @@ public TagHelperTest(ApplicationWithTagHelpersFixture fixture)
 
         public ApplicationTestFixture Fixture { get; }
 
-        public static TheoryData ApplicationWithTagHelpersData
+        public static IEnumerable<object[]> ApplicationWithTagHelpersData
```

Comment:
Not the same. I'll re-do how it works to be equivalently `TheoryData<string, RuntimeFlavor>`. This was pre-existing code 😉